src/Model/Filter/Value.php

```diff
@@ -27,6 +28,16 @@ public function process()
             return false;
         }
 
+        if (!$this->manager()->getRepository() instanceof Table) {
```

Comment:
Shouldn't this block go after the multi values check below?

Reply:
Let me check

Reply:
Changed :smile: